service/src/main/kotlin/io/provenance/explorer/domain/entities/Blocks.kt

```diff
@@ -308,3 +266,62 @@ class MissedBlocksRecord(id: EntityID<Int>) : IntEntity(id) {
     var runningCount by MissedBlocksTable.runningCount
     var totalCount by MissedBlocksTable.totalCount
 }
+
+object BlockCacheHourlyTxCountsTable : IntIdTable(name = "block_cache_hourly_tx_counts") {
```

Comment:
See above note about adding `id` to the table, the `IntIdTable` already references an id, so no changes needed here.

Reply:
For using the timestamp as the id, heres how it would go 

```
object BlockCacheHourlyTxCountsTable : IdTable<DateTime>(name = "block_cache_hourly_tx_counts") {
    override val id = BlockCacheHourlyTxCountsTable.datetime("block_timestamp").entityId()
    val blockTimestamp = datetime("block_timestamp")
    val txCount = integer("tx_count")
}

class BlockCacheHourlyTxCountsRecord(id: EntityID<DateTime>) : Entity<DateTime>(id) {
    companion object : EntityClass<DateTime, BlockCacheHourlyTxCountsRecord>(BlockCacheHourlyTxCountsTable) {
```

Reply:
Perfect! `block_timestamp` is indexed but should we make it a unique index?